hs_discover/src/components/Search.vue

```diff
@@ -184,24 +183,41 @@
             </div>
         </div>
         <div id="resource-rows" class="col-lg-9">
-            <br/>
-            <div class="table-wrapper">
-              <p class="table-message" style="color:red" v-if="(!resources.length) && (authorFilter.length ||
-              ownerFilter.length || subjectFilter.length || contributorFilter.length || typeFilter.length ||
-              availabilityFilter.length || searchtext !== '' || startdate !== '' || enddate !== '')"><i>No resource matches</i></p>
+          <div class="row" style="padding-right:14px">
+
+                  <div v-if="resources.length > 0" id="resultsdisp-upper">
+            <span id="page-left-upper" style="cursor:pointer" v-on:click="paging(-1)" v-b-tooltip.hover title="Go back a page"
+                class="pagination fa fa-angle-double-left fa-w-14 fa-fw fa-2x interactive"></span>
+            Page <input v-b-tooltip.hover title="Enter number or use keyboard up and down arrows" id="page-number-upper"
+                    type="number" v-model="pagenum" @change="searchClick(true)" min="1" :max="pagecount"> of {{pagecount}}
+            <span id="page-right-upper" style="cursor:pointer" v-on:click="paging(1)" v-b-tooltip.hover title="Go forward a page"
+                class="pagination fa fa-angle-double-right fa-w-14 fa-fw fa-2x interactive"></span>
+                Resources {{Math.max(0, pagedisp * perpage - perpage + 1)}} - {{Math.min(rescount, pagedisp * perpage)}} of {{rescount}}
+                  </div>
+
+            <div class="table-wrapper" style="overflow: auto">
+
                 <table id="items-discovered" v-if="resources.length"
                     class="table-hover table-striped resource-custom-table main-table">
                     <thead>
-                        <tr><th><!-- placeholder --></th>
-                            <th v-for="key in labels" v-bind:key="key" style="cursor:pointer"
-                                @click="sortBy(key)">
-                                <i :class="sortStyling(key)"></i>{{key}}
+                        <tr><th class="tbl-col-icons"><!-- placeholder --></th>
```

Comment:
Instead of dynamic Vue table headers, use explicit headers, allowing assignment of classes with set widths. This is part of ongoing responsive design